recipes/fluidimage/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "fluidimage" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f2910a799bcd625a3a10ae6afb3e05668629e21764ea6ee171774a0817afee00
+
+build:
+  number: 0
+  entry_points:
+    - fluidimviewer-pg = fluidimage.gui.pg_main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - setuptools_scm
+    - transonic
+    - pythran
+  run:
+    - dask
+    - fluiddyn >=0.2.0
+    - h5netcdf
+    - h5py
+    - imageio
+    - matplotlib >=1.4.2
+    - numpy >=1.8
```

Are you using the NumPy C API? If you are you should add `numpy` to `host` and change this to:
```suggestion
    - {{ pin_compatible('numpy') }}
```